apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/variables/[variable]/index.vue

```diff
@@ -0,0 +1,151 @@
+<script setup lang="ts">
```

Comment:
This is nice. I think main power of this card would be to navigate harmonization efforts
<img width="1401" alt="image" src="https://github.com/molgenis/molgenis-emx2/assets/120060/778dae74-2f55-477d-a465-b98e693dd0eb">

A challenge will be that users overall will want to have filters on the network/cohorts they would like to include in this view also. For now we can accept that we simply will see all and instead focus on the central harmonization overview to provide such navigation.

Reply:
note 100% sure about the userflow / user task here , bug agree to get the setup up and running and then maybe add additional local ( part of page ) filter options , or create dedicated ui to support this task